### PR TITLE
Revert "Revert "Rdm 13033""

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by danlysiak — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -264,8 +264,8 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.14.0'
-    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.2-Categories'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.13.5-prerelease'
```
Comment (MSancaktutar) [inline build.gradle:267]: Please let's use a latest stable release.

File: build.gradle [hunk 2]
```diff
@@ -264,8 +264,8 @@ dependencies {
...
+    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.1'
```
Comment (MSancaktutar) [inline build.gradle:268]: Would this now lose the specific delta in 8.7.2-Categories? I'd suggest releasing '8.7.2-Categories' into a next stable release and use that one, if I'm not getting it wrong.

File: dependency-check-suppressions.xml [hunk 3]
```diff
@@ -49,4 +49,17 @@
...
+		<cve>CVE-2021-44832</cve>
+		<cve>CVE-2021-42550</cve>
+		<cve>CVE-2021-22060</cve>
+		<cve>CVE-2022-23181</cve>
+	</suppress>
+
+	<suppress>
+		<packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-core@.*$</packageUrl>
+		<vulnerabilityName>CWE-862: Missing Authorization</vulnerabilityName>
+	</suppress>
```
Comment (MSancaktutar) [inline dependency-check-suppressions.xml:63]: Well, I wouldn't think new suppressions would be needed, as the master build was very recently green.

**Review by MSancaktutar — APPROVED** [review]
Would be good to action 2 comments about befta and test-definitions dependencies, otherwise good. As discussed, would be good to make sure the fix in the contract isn't breaking any client code that was meant to tolerate the broken aspect of the contract.

**Pull request merged**